x-pack/plugins/transform/public/app/sections/transform_management/components/edit_transform_flyout/edit_transform_flyout_form.tsx

```diff
@@ -33,36 +33,130 @@ export const EditTransformFlyoutForm: FC<EditTransformFlyoutFormProps> = ({
           defaultMessage:
             'The interval between checks for changes in the source indices when the transform is running continuously. Also determines the retry interval in the event of transient failures while the transform is searching or indexing. The minimum value is 1s and the maximum is 1h.',
         })}
         label={i18n.translate('xpack.transform.transformList.editFlyoutFormFrequencyLabel', {
           defaultMessage: 'Frequency',
         })}
         onChange={(value) => dispatch({ field: 'frequency', value })}
-        placeholder="1m"
+        placeholder={i18n.translate(
+          'xpack.transform.transformList.editFlyoutFormFrequencyPlaceholderText',
+          {
+            defaultMessage: 'Default: {defaultValue}',
+            values: { defaultValue: formFields.frequency.defaultValue },
+          }
+        )}
         value={formFields.frequency.value}
       />
+
+      <EuiSpacer size="l" />
+
+      <EuiAccordion
+        data-test-subj="transformEditAccordionDestination"
+        id="transformEditAccordionDestination"
+        buttonContent={i18n.translate(
+          'xpack.transform.transformList.editFlyoutFormDestinationButtonContent',
+          {
+            defaultMessage: 'Destination configuration',
+          }
+        )}
+        paddingSize="s"
+      >
+        <EditTransformFlyoutFormTextInput
+          dataTestSubj="transformEditFlyoutDestinationIndexInput"
+          errorMessages={formFields.destinationIndex.errorMessages}
+          label={i18n.translate(
+            'xpack.transform.transformList.editFlyoutFormDestinationIndexLabel',
+            {
+              defaultMessage: 'Destination index',
+            }
+          )}
+          onChange={(value) => dispatch({ field: 'destinationIndex', value })}
+          value={formFields.destinationIndex.value}
+        />
+
+        <EditTransformFlyoutFormTextInput
+          dataTestSubj="transformEditFlyoutDestinationPipelineInput"
+          errorMessages={formFields.destinationPipeline.errorMessages}
+          label={i18n.translate(
+            'xpack.transform.transformList.editFlyoutFormDestinationPipelineLabel',
+            {
+              defaultMessage: 'Pipeline',
+            }
+          )}
+          onChange={(value) => dispatch({ field: 'destinationPipeline', value })}
+          value={formFields.destinationPipeline.value}
+        />
+      </EuiAccordion>
+
+      <EuiSpacer size="l" />
+
+      <EuiAccordion
+        data-test-subj="transformEditAccordionAdvancedSettings"
+        id="transformEditAccordionAdvancedSettings"
+        buttonContent={i18n.translate(
+          'xpack.transform.transformList.editFlyoutFormAdvancedSettingsButtonContent',
+          {
+            defaultMessage: 'Advanced settings',
+          }
+        )}
+        paddingSize="s"
+      >
+        <div data-test-subj="transformEditAccordionAdvancedSettingsContent">
+          <EditTransformFlyoutFormTextInput
```

Comment:
Should we allow `0` here? The default value for `docs_per_second` is `null` according to the docs, but with this form there doesn't seem to be any way of setting it back to the default.

Reply:
Fixed the code to correctly pass nullable fields on as `null` in 3380cbfdcb0.